R/plot_corr.R

```diff
@@ -19,7 +19,7 @@ plot_corr <- function(data, vrb = "all", label = FALSE, square = TRUE, diagonal
   if (vrb[1] == "all") {
     vrb <- names(data)
   } else {
-    vrb <- names(select(data, all_of({{vrb}})))
+    vrb <- names(dplyr::select(data, all_of({{vrb}})))
```

Comment:
Could you please add [`tidyselect::`](https://www.rdocumentation.org/packages/tidyselect/versions/1.2.0/topics/all_of) before all of the `all_of(...)` calls for clarity/completeness?